README.md

```diff
@@ -7,10 +7,10 @@ BYAR mutator for Chobby
 - [Linux](https://spring-launcher.ams3.digitaloceanspaces.com/beyond-all-reason/BYAR-Chobby/Beyond%20All%20Reason.AppImage)
 - [Windows](https://spring-launcher.ams3.digitaloceanspaces.com/beyond-all-reason/BYAR-Chobby/Beyond%20All%20Reason.exe)
 
-# Development setp
+# Development steps
 
 1. Download using one of the links above and start the application.
 2. Start the application once using the "Test" version and launch the lobby. This will download the engine and other dependencies.
 3. Open the install folder (there's a button for that in the launcher), and clone this repository inside the `games` folder, `git clone https://github.com/beyond-all-reason/BYAR-Chobby.git BYAR-Chobby.sdd`
 4. Choose the "Dev Lobby" config which will run the lobby version on your PC. You can now develop and test any BYAR specific lobby functionality.
-5. If you want to develop Chobby itself, clone it in the `games` folder similar to step 3: `git clone https://github.com/Spring-Chobby/Chobby.git Chobby.sdd`. You must also change the `depend` table in `BYAR-Chobby.sdd/modinfo.lua` to use `Chobby $VERSION` instead of `rapid://chobby:test`.
\ No newline at end of file
+5. If you want to develop Chobby itself, clone it in the `games` folder similar to step 3: `git clone https://github.com/Spring-Chobby/Chobby.git Chobby.sdd`. You must also change the `depend` table in `BYAR-Chobby.sdd/modinfo.lua` to use `Chobby $VERSION` instead of `rapid://chobby:test`.
```

Comment:
When I edited this readme from github I only edited line 10 but it auto edited the last line of the readme (line 16). I tried download this modified readme and I saw an additional line (line 17) was added. Was like "github" auto added a line at the end of the readme and it's maybe why line 16 changed. (I tried edit only one line of another file from github (not a readme file) and I had not this behavior that a supplementary line was edited, like for this readme.)